bpf/bpf_sock.c

```diff
@@ -676,6 +703,19 @@ static __always_inline void ctx_get_v6_src_address(const struct bpf_sock *ctx,
 }
 #endif /* ENABLE_NODEPORT */
 
+static __always_inline void ctx_get_v6_dst_address(const struct bpf_sock *ctx,
```

Comment:
The tests are failing with a compilation error because this function is defined even when IPv6 is disabled. The easiest way to fix this might be to extend the `ENABLE_IPV6` `ifdef` to cover all the `ctx_{get,set}` functions for IPv6, until `ctx_set_v6_address`.